documentation/style-guide.rst

```diff
@@ -102,6 +102,44 @@ Unix
    The name of the operating system developed at AT&T Bell Labs in the early
    1970s.
 
+Diátaxis
+========
+
+Python's documentation follows the `Diátaxis <https://diataxis.fr//>`_
```

Comment:
I forgot to include this in my first review: this should definitely be something like: *Python's documentation is moving towards an implementation of Diátaxis.* It's not there yet by any means!

Reply:
Maybe something like "strives to follow", so we don't need to update the text once we do get there?